test/parallel/test_torch.py

```diff
@@ -2088,17 +2087,16 @@ def check_file(fname, check_cycle=True):
                 with open(fname, 'r') as timeline_file:
                     timeline_text = timeline_file.read()
                     assert 'allreduce.test_allreduce' in timeline_text, timeline_text
-                    assert 'NEGOTIATE_ALLREDUCE' in timeline_text, timeline_text
-                    assert 'ALLREDUCE' in timeline_text, timeline_text
                     assert 'start_time_since_epoch_in_micros' in timeline_text, timeline_text
+                    assert 'NEGOTIATE_ALLREDUCE' in timeline_text, timeline_text
```

Comment:
Glad we got this one working again.  Shouldn't `ALLREDUCE` be in here as well?

Reply:
Yes. Thanks. Done.